### PR TITLE
(RE-5953) Disable gpgcheck for el4

### PR DESCRIPTION
**Pull request by underscorgan**

### CODE REVIEW

**Comment by underscorgan** [comment]
```
[pl-build-tools]
name=Puppet Labs Build Tools Repository el 4 - $basearch
baseurl=http://pl-build-tools.delivery.puppetlabs.net/yum/el/4/$basearch
gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-pl-build-tools
enabled=1
gpgcheck=0

#[pl-build-tools-source]
#name=Puppet Labs Build Tools Repository el 4 - Source
#baseurl=http://pl-build-tools.delivery.puppetlabs.net/yum/el/4/SRPMS
#gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-pl-build-tools
#failovermethod=priority
#enabled=0
#gpgcheck=0
```

**Comment by MikaelSmith** [comment]
Seems fair. :+1:

**Comment by branan** [comment]
not that bad

File: configs/platforms/el-4-i386.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@
   plat.defaultdir "/etc/sysconfig"
   plat.servicetype "sysv"
 
-  plat.provision_with "yum install -y --nogpgcheck autoconf automake createrepo rsync gcc make rpmdevtools rpm-libs yum-utils rpm-sign rpm-build; echo '[build-tools]\nname=build-tools\nbaseurl=http://enterprise.delivery.puppetlabs.net/build-tools/el/4/$basearch\ngpgcheck=0' > /etc/yum.repos.d/build-tools.repo"
+  plat.provision_with "yum install -y autoconf automake createrepo rsync gcc make rpmdevtools rpm-libs yum-utils rpm-sign rpm-build; echo '[build-tools]\nname=build-tools\nbaseurl=http://enterprise.delivery.puppetlabs.net/build-tools/el/4/$basearch\ngpgcheck=0' > /etc/yum.repos.d/build-tools.repo"
   plat.install_build_dependencies_with "yum install -y --nogpgcheck"
```
Comment (geoffnichols) [inline configs/platforms/el-4-i386.rb]: Do we need to remove this `--nogpgcheck` as well?

**Comment by shrug** [comment]
njnd26r

File: configs/platforms/el-4-x86_64.rb [hunk 2]
```diff
@@ -3,7 +3,7 @@
   plat.defaultdir "/etc/sysconfig"
   plat.servicetype "sysv"
 
-  plat.provision_with "yum install -y --nogpgcheck autoconf automake createrepo rsync gcc make rpmdevtools rpm-libs yum-utils rpm-sign rpm-build; echo '[build-tools]\nname=build-tools\nbaseurl=http://enterprise.delivery.puppetlabs.net/build-tools/el/4/$basearch\ngpgcheck=0' > /etc/yum.repos.d/build-tools.repo"
+  plat.provision_with "yum install -y autoconf automake createrepo rsync gcc make rpmdevtools rpm-libs yum-utils rpm-sign rpm-build; echo '[build-tools]\nname=build-tools\nbaseurl=http://enterprise.delivery.puppetlabs.net/build-tools/el/4/$basearch\ngpgcheck=0' > /etc/yum.repos.d/build-tools.repo"
   plat.install_build_dependencies_with "yum install -y --nogpgcheck"
```
Comment (geoffnichols) [inline configs/platforms/el-4-x86_64.rb]: Do we need to remove this `--nogpgcheck` as well?

File: configs/platforms/el-4-i386.rb (hunk 1 above)
Comment (underscorgan) [inline configs/platforms/el-4-i386.rb]: `--nogpgcheck` is an invalid option on  el4. Everything is awesome.

Comment (underscorgan) [inline configs/platforms/el-4-i386.rb]: Hm. It succeeded with just the first one, but probably should.

Comment (haus) [inline configs/platforms/el-4-i386.rb]: puppetlabs-release probably *has* no build dependencies, which i imagine is why it succeeded.

**Comment by underscorgan** [comment]
@geoffnichols all better!

**Pull request merged**